Web/starter_project_vanguard/pages/_app.tsx

```diff
@@ -1,6 +1,8 @@
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
 
+import Layout from '@/components/layout'
+
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return <Layout><Component {...pageProps} /> </Layout>
```

Comment:
is there a way to decide which layout to use for each page? (incase we end up making multiple layouts)
like for index page I could want a layout that has a footer in it, for login I may want one that has only navbar or sth

I just want you to check if there is an existing way to deal with layouts in NextJS but for now this should work fine too

Reply:
Yes, there is. but it isn't necessary for this project as all the pages have the same footer and header.